weave-js/src/components/PagePanelComponents/Home/Browse3/pages/wfReactInterface/interface.ts

```diff
@@ -297,7 +416,17 @@ const typeNameToCategory = (typeName: string): ObjectCategory | null => {
   return null;
 };
 
-const objectArtifactVersionNodeToDictNode = (
+const opNameToCategory = (opName: string): OpCategory | null => {
+  const categories = ['train', 'predict', 'score', 'evaluate', 'tune'];
```

Comment:
I think we should move this up and rework `type OpCategory` so there's a single source of truth.

Reply:
👍